Readme.md

```diff
@@ -29,6 +31,40 @@ export const digestStream = async (source: AsyncIterable<Uint8Array>) => {
   // memory manually once you're done.
   hasher.free()
 
+  // Return subarray containing multihash bytes
+  return digest.subarray(0, offset)
+}
+```
+
+You could also know precise number of bytes that will be needed to hold the multihash once you're done feeding bytes into it
```

Comment:
The trade off is that this is slightly slower because another round trip to WASM? Can we document if so?

Reply:
No tradeoff is you don't know that until your done doing writes, which may not be a good fit for the case where you slab allocate buffer that will hold this bytes along with other things. In this case you probably want to go with `Hasher.MAX_SIZE` and move cursor by number of bytes written eventually trimming the end.